ci/setup-rust.sh

```diff
@@ -0,0 +1,14 @@
+#!bash
+set -eux
```

Comment:
Can you add a comment to explain under which context this script is going to be use

Reply:
done